### PR TITLE
File Service Plugin

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -359,5 +359,29 @@
...
+    "plugin_file_service_display_name": "插件: 文檔服務",
+    "plugin_file_service_file_key_copied": "已拷貝 FileKey",
+    "plugin_file_service_on_insert": "使用此文檔",
+    "plugin_file_service_uploading_preparing": "準備上傳……",
+    "plugin_file_service_uploading_estimating_time": "正在計算剩餘時間",
+    "plugin_file_service_uploading_in_minute_remaining": "剩餘 {{seconds}} 秒",
+    "plugin_file_service_uploading_remaining": "剩餘 {{minutes}} 分 {{seconds}} 秒",
+    "plugin_file_service_uploading_on_cancal": "上傳中，請勿關閉頁面",
+    "plugin_file_service_show_more": "更多",
+    "plugin_file_service_file_key": "File Key:",
```
Comment (guanbinrui) [inline src/_locales/zh/messages.json:372]: This field need not translate?

File: src/utils/type-transform/String-ArrayBuffer.ts [hunk 2]
```diff
@@ -1,19 +1 @@
-export function encodeText(str: string) {
```
Comment (guanbinrui) [inline src/utils/type-transform/String-ArrayBuffer.ts:1]: Remove this file and import those functions from the package directly.

**Review by guanbinrui — APPROVED** [review]

File: src/_locales/zh/messages.json (hunk 1 above)
Comment (septs) [reply src/_locales/zh/messages.json:372]: CC @neruthes

File: src/utils/type-transform/String-ArrayBuffer.ts (hunk 2 above)
Comment (septs) [reply src/utils/type-transform/String-ArrayBuffer.ts:1]: minimize changes

File: src/_locales/zh/messages.json (hunk 1 above)
Comment (neruthes) [reply src/_locales/zh/messages.json:372]: I believe that keeping English here is better, at least for now.

**Comment by Jack-Works** [comment]
don't merge now, I'm still reviewing

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**